### PR TITLE
feat(module:popconfirm): support nzAutoFocus

### PR DESCRIPTION
**Pull request by yangjunhan**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #6249 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: components/tooltip/base.ts [hunk 1]
```diff
@@ -359,7 +359,7 @@ export abstract class NzTooltipBaseComponent implements OnDestroy, OnInit {
 
   _positions: ConnectionPositionPair[] = [...DEFAULT_TOOLTIP_POSITIONS];
 
-  private destroy$ = new Subject<void>();
+  destroy$ = new Subject<void>();
```
Comment (wendellhu95) [inline components/tooltip/base.ts:362]: Make this property as `protected`.

**Review by vthinkxie — CHANGES_REQUESTED** [review]
Hi @yangjunhan
@wendellhu95 has left some comments, could you have a look?
thanks

File: components/popconfirm/popconfirm.ts [hunk 2]
```diff
@@ -190,6 +201,21 @@ export class NzPopconfirmComponent extends NzToolTipComponent implements OnDestr
     super(cdr, directionality, noAnimation);
   }
 
+  ngAfterViewInit(): void {
+    combineLatest([this.okBtn.changes, this.cancelBtn.changes])
```
Comment (hsuanxyz) [inline components/popconfirm/popconfirm.ts:205]: use cdk/focusTrap

**Review by hsuanxyz — CHANGES_REQUESTED** [review]

File: components/tooltip/base.ts (hunk 1 above)
Comment (yangjunhan) [reply components/tooltip/base.ts]: done

File: components/popconfirm/popconfirm.ts (hunk 2 above)
Comment (yangjunhan) [reply components/popconfirm/popconfirm.ts]: done

**Review by hsuanxyz — APPROVED** [review]
LGTM

**Pull request merged**